scripts/data_overlap/ngram_hasher.py

```diff
@@ -0,0 +1,121 @@
+from typing import List, Tuple, Dict
+
+# A prime number chosen as a base for exponentiation, larger than 171,476 (estimated vocab size of English words)
+RABIN_KARP_PRIME = 172001
+
+# A prime number chosen as a base for hashing characters, larger than 256 (UTF-8 character space)
```

Comment:
256 is the ASCII character space, not the UTF-8 characters space. Is this choice of prime still valid for the UTF-8 character space?

Reply:
Yes, the number was more of a heuristic